app/services/edtf_date.rb

```diff
@@ -12,4 +12,10 @@ def self.valid?(str)
 
     (regular_date || edtf_date).present?
   end
+
+  def self.humanize(str)
+    Date.edtf(str)&.humanize || str.to_s
+  rescue RuntimeError
+    str.to_s
+  end
```

Comment:
This should ensure that the migrated pubished dates, which may not be valid EDTF, will display, right?